hasktorch/src/Torch/Typed/Native.hs

```diff
@@ -2338,8 +2338,16 @@ q_zero_point _input = unsafePerformIO $ (cast1 ATen.q_zero_point_t) _input
 -- combinations :: Tensor dtype shape -> Int -> Bool -> Tensor dtype shape
 -- combinations _input _r _with_replacement = unsafePerformIO $ (cast3 ATen.combinations_tlb) _input _r _with_replacement
 
--- lstm_cell :: Tensor dtype shape -> [Tensor dtype shape] -> Tensor dtype shape -> Tensor dtype shape -> Tensor dtype shape -> Tensor dtype shape -> (Tensor dtype shape,Tensor dtype shape)
--- lstm_cell _input _hx _w_ih _w_hh _b_ih _b_hh = unsafePerformIO $ (cast6 ATen.lstm_cell_tltttt) _input _hx _w_ih _w_hh _b_ih _b_hh
+lstm_cell :: Tensor dtype '[batchSize, inputDim] 
+  -> [Tensor dtype '[batchSize, hiddenSize]]
```

Comment:
I believe it would be better if this could be a `Tensor dtype '[seqLen, batchSize, hiddenSize]` instead of a list of tensors, what do you think?

Reply:
The `lstm_cell` does not manage recurrent application, only state transitions.  As such, it only knows about `batchSize`, `inputDim`, and `hiddenSize`.  The list due to the underlaying c++ api which expects to receive the current state value as a list.